go/runtime/client/tests/tester.go

```diff
@@ -146,6 +146,13 @@ func testQuery(
 	// Check for values from TestNode/Client/SubmitTx
 	require.EqualValues(t, testInput, txns[0])
 
+	// Check events query (see mock worker for emitted events).
+	events, err := c.GetEvents(ctx, &api.GetEventsRequest{RuntimeID: runtimeID, Round: 2})
```

Comment:
Can you also update the txsource runtime workload to query the events?

Reply:
Added.